Resources/Prototypes/_Goobstation/Heretic/Roles/Antags/heretic.yml

```
@@ -4,4 +4,11 @@
   setPreference: true
   objective: roles-antag-heretic-description
-  guides: [ Heretics ]
\ No newline at end of file
+  guides: [ Heretics ]
+
+- type: startingGear
+  id: HereticGear
+  storage:
+    back:
+    - CodexCicatrix
+    - OrganHumanHeart
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)